### DISCUSSION TITLE
v1.9.0

### DISCUSSION THREAD

**Discussion by jjw24** [Flow-Launcher/Flow.Launcher#882] [Announcements]
Release Notes
=============

This release contains some major redesigns to Flow's main search and settings window UIs, we would like to say a big thank you to @onesounds for the tremendous work in modernising Flow's look and feel. Please see below for the highlights and full change log.

## Highlights

- All new UI design

- New themes

- New Settings window

- Animation & sound effect for search window

- Light/Dark/Windows System Default color

- New Plugin Store

- Game Mode for when you are playing games

- Wizard window for beginners

- Python plugins can now have settings in the Settings window

- Global browser and explorer settings

- Crowdin support for translations

## Features

- Global browser setting with private mode option

- Python plugin setting support for the settings window

- Added Portuguese (Portugal) Language 

- Added tab to autocomplete result 

- Explorer plugin now supports autocomplete of directory/file path

- Wizard window for our beginners 

- Added Glyph Icon support 

- Add Quick Access action keyword for Explorer 

- Add Crowdin Integration

- Copy to clipboard is available via the API for plugins 

- Added Flow's version data as environmental vars that can be accessed by Python plugins 

- Switched off the default launch on system start up, so Flow will not start up when the computer starts by default 

- You can now hide app's path for Program plugin 

- Game Mode added for those that would like to pause Flow during their gaming 

- Manual plugin installation 

- Animation and sound when triggering Flow 

- Default theme is changed 

- Settings control template updated 

- Updated Slovak translations

- Default File manager in settings that can be switched to use different explorers globally 

- Tweaked translation api 

- the Tray Menu is redesigned kind/ui 

- For latest Windows release, Flow's popup now uses Windows notification 

- Plugins can now execute shell commands via Flow's API 

- New WindowsSettings plugin, replacing ControlPanel plugin 

- New Plugin Store 

- Tweaked language functionality 

- Added modern themes 

- Support  download plugins via CDN 

- Refactor Bookmark plugin 

- Added Advanced Boot Options restart for Sys plugin 

- Added run as administrator when holding ctrl and shift 

- Bumped applicable nuget packages to latest versions 

- Refactor JsonRPC structure 

- Updated Embedded Python download message from install to download 

- Cache program result
 
## Bug fixes

- Fixed misaligned suggestion text when query is long, especially in WebSearch plugin 

- Fixed overlapping of scrollbar and resize area

- Fixed Typo TermSeparator & removed the actionkeyword in Terms (API) 

- Make suggestions hidden if query matches 

- Refactored Shell Command

- Fixed HideonStartup

- Adjusted WebSearch plugin 

- Added width in setting and fixed max result displayed not reflected in real-time 

- Fix empty query issue 

- Tweaked result update functionality 

- Fixed Explorer plugin crash on semicolon 

- Fixed WinGet update CI 

- Implemented re-query option and added re-query when calling customQueryHotkey 

## Contributions
We would like to directly mention and thank the following people (in alphabetical order) for their contribution and support in this release, making Flow even better as this year wraps up.

@deefrawley, @Garulf, @JohnTheGr8, @kubalav, @onesounds, @pc223, @spedrickson, @taooceros, @themike10452, @Zeroto521 and everyone who has joined discussions, put through their ideas or reported bugs.

<hr /><em>This discussion was created from the release <a href='https://github.com/Flow-Launcher/Flow.Launcher/releases/tag/v1.9.0'>v1.9.0</a>.</em>

**Comment by comradekingu** [comment]
Happy to help if you use Weblate instead of Crowdin.
